src/main/java/net/minecraftforge/event/world/ChunkDataEvent.java

```diff
@@ -84,7 +91,7 @@ public Load(ChunkAccess chunk, CompoundTag data, ChunkStatus.ChunkType status)
     /**
      * ChunkDataEvent.Save is fired when vanilla Minecraft attempts to save Chunk data.<br>
      * This event is fired during chunk saving in
-     * {@link AnvilChunkLoader#saveChunk(World, Chunk)}. <br>
+     * {@code ChunkMap#save(ChunkAccess)}. <br>
```

Comment:
Code inconsistency.

Reply:
In what way? From what I can tell, this is the replacement for this method.